src/web_app/css/main.css

```diff
@@ -104,6 +104,22 @@ footer > div.active {
   opacity: .8;
 }
 
+div#privacy {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  /* Makes sure to stay behind room links so they can be clicked */
+  z-index: -1
```

Comment:
I do not think this is the way to go since we cannot add it to the existing footer as per the comment below, do you have other suggestions for improvement?